### ISSUE TITLE
add notification tabs for different classes of notifications

### ISSUE THREAD

**Issue by cfcs** [hannesm/jackline#174]
for example, it would be super nice to have "subscription requested" messages in a dedicated tab, I find I often overlook them in the main tab.

**Comment by hannesm** [comment]
I agree that something should be done here.  The behaviour changed as a side effect of fixing https://github.com/hannesm/jackline/issues/163
